### DISCUSSION TITLE
deploy-nestjs-cloud-run

### DISCUSSION THREAD

**Discussion by giscus** [tomwray13/tomray.dev#47] [Blog Comments]
# deploy-nestjs-cloud-run

Learn how to deploy a NestJs app with continuous deployments to Cloud Run using Github Actions and Cloud Build

https://www.tomray.dev/deploy-nestjs-cloud-run#create-a-service-account-with-permissions

**Comment by phsantiago** [comment]
This article was super helpful!
Used to spin up two different environments already

**Reply by tomwray13** [reply]
Thank you @phsantiago!

**Comment by mozzipa** [comment]
Thanks for very helpful instruction!
But mine is not being updated after first deploy.
Can you advise me way to have updating for every single git push?

**Reply by mozzipa** [reply]
It was able with your instruction provided that pulling of .github/workflow/*.yml prior to git push.
Thanks.

**Comment by SerhiiBabich** [comment]
Hello, can you please make the same great article, only for the implementation of github actions as change checks (ci)

**Comment by ayodeji1167** [comment]
If my application needs some env variables on start... how do i pass that in the github workflow file

**Comment by SaadBazaz** [comment]
Why not just use GCP's built-in Build Trigger? You can simply log into your GitHub account and connect your repo, and it'll automatically build your selected branch.

https://cloud.google.com/run/docs/continuous-deployment-with-cloud-build

What benefits does your approach give over that?
